src/app/components/form/form.component.ts

```diff
@@ -76,6 +94,9 @@ export class FormComponent implements OnInit, OnChanges, OnDestroy {
 
   ngOnDestroy() {
     this.langChangeSubscription.unsubscribe();
+    if ( this.routeParamsSubscription ) {
```

Comment:
Is there a case where this is not defined?

Reply:
Not that I'm aware of, I removed the test.